plugin/src/main/kotlin/org/jetbrains/intellij/versions/KotlinPluginVersion.kt

```diff
@@ -0,0 +1,32 @@
+package org.jetbrains.intellij.versions
+
+@Suppress("ClassName")
+sealed class KotlinPluginVersion(val value: String, val location: String): java.io.Serializable {
```

Comment:
What should be supported here:
* short plugin notation like "1.2.60" or URL plugin notation (explicitly give URL to download)
* conversion of IDEA version + Kotlin version to full Kotlin version like "1.2.60-release-IJ2018.1-1"
* map of full Kotlin version to URL (I'd add last versions like 1.2.51, 1.2.50, 1.2.41, 1.2.40)
* if explicit URL is used, compatibility with current IDEA should be checked (using downloaded file name). For example, something with name `***-release-IJ2018.1-*` can be used on IDEA 2018.1 (or 2018.1.*, or 181.*) only